mlflow/projects/__init__.py

```diff
@@ -212,7 +212,7 @@ def run(
     :param experiment_name: Name of experiment under which to launch the run.
     :param experiment_id: ID of experiment under which to launch the run.
     :param backend: Execution backend for the run: MLflow provides built-in support for "local",
-                    "databricks", and "kubernetes" (experimental) backends. If running against
```

Comment:
Let's keep the Kubernetes backend as `experimental` until we've fixed some of the reported issues.